src/types.rs

```diff
@@ -6,9 +6,39 @@ use ipnetwork;
 use serde::{de::Error, Deserialize, Deserializer, Serialize, Serializer};
 use thiserror::Error;
 
+#[derive(Clone, PartialEq, Eq, Deserialize, Serialize)]
+pub struct AccountId(String);
+
+impl AccountId {
+    pub fn from_string_unchecked(id: String) -> Self {
+        AccountId(id)
+    }
+
+    // The digit portion of the account hint.
+    pub fn hint(&self) -> &str {
+        &self.0[0..3]
```

Comment:
sry, I missed this before approval. Combined with the unchecked initialization this can panic for a couple of reasons. I don't trust our JS code enough to risk it.
`self.0.get(0..3).unwrap_or_default()` should do it I guess?

Reply:
Ah yes, for some reason I thought that slices like this rounded down as long as they started in range. I'll send a PR.